config.ru

```diff
@@ -1,10 +1,13 @@
 require './app'
 
+set :database, ENV['DATABASE_URL'] || 'postgres://localhost/[YOUR_DATABASE_NAME]'
```

Comment:
This doesn't look like a bad line of code, but it doesn't have anything to do with this request.  See if you can get rid of it.